articles/101_build_tool.md

```diff
@@ -289,59 +291,142 @@ The following items are possible extension points to provide custom functionalit
+The separation of the build tool name from the supported build systems as well as the separation from being a "ROS-only" tool will hopefully also help users to understand the difference and attract new users and potential contributors.
 
-### Start "from scratch"
+### Next steps
 
-Another approach is to implement the necessary software architecture to enable the desired flexibility and modularity "from scratch".
-Then fill in the features step-by-step by porting existing building blocks from the existing solutions.
-Some items to highlight the necessary efforts (not a complete list):
+The following next steps will happen before the next ROS 2 release *Bouncy*.
 
-- Create the software architecture to support the flexibility sketched by the extension points listed above which will be easier "from scratch" than for an existing code base.
+- The instructions in the ROS 2 wiki to build from source will be updated to use `colcon` instead.
+- The `ament_tools` repository will be archive, removed from the `ros2.repos` file, and won't be released into *Bouncy*.
+- The ROS 2 buildfarm(s) will be updated to use `colcon` and provide devel / PR / prerelease jobs for the *Bouncy* release.
 
-- Port / reimplement many of the features existing in the other build tools.
-  It will take a non trivial amount of time to reach the feature level of e.g. `catkin_tools`
+### Implications
 
-- Thorough test the functionality and write documentation for developers as well as users.
+The following items briefly enumerate what This means for ROS developers and users:
 
-## Proposal
+- **No CMake code** of any ROS 2 (or ROS 1) package **needs to be changed** for this transition.
+- When building and testing ROS 2 the command `colcon build` / `colcon test` will be used instead of `ament build` / `ament test`.
+  Please see the [documentation](http://colcon.readthedocs.io/en/latest/migration/ament_tools.html) how to map `ament_tools` command line arguments to `colcon` arguments.
+- For ROS 1 nothing is changing at this point in time.
 
-The decision of which approach should be selected is deferred at the moment since nobody is likely able to spend any time on this in the foreseeable future.
-It could also be considered an implementation detail - as long as the described goals are reached it doesn't matter how the unified build tool was getting there.
+### Outlook
 
-*Hopefully to be continued in the future...*
+- Since `colcon` can be used to build ROS 1 early adopters can try to use it to build ROS 1 from source.
+  While there is documentation how to migrate from [catkin_make_isolated](http://colcon.readthedocs.io/en/latest/migration/catkin_make_isolated.html) and [catkin_tools](http://colcon.readthedocs.io/en/latest/migration/catkin_tools.html) `colcon` won't be the recommended build tool in ROS 1 for the foreseeable future.
```

Comment:
But the implication is that `colcon` will eventually be recommended instead of `catkin_make_isolated` and `catkin_make`, and that `catkin_tools` will not be receiving attention with respect to ROS 2. And because `colcon` will eventually be the recommendation, we will therefore not spend any extra time on `catkin_tools` for ROS 1 either.  Is that not the case?

Reply:
Yes, that is the case.